.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,28 @@
+name: Dependabot auto-merge
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  dependabot-check:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1.1.1
```

Comment:
Hi @DnsZhou , just to let you know, this is wrong (very old) version, and `if` condition below is also failing

We need to double-check those things during code reviews, were you able to test it?